### DISCUSSION TITLE
Autogen 0.2, Azure 4o compatability

### DISCUSSION THREAD

**Discussion by LasseTS** [microsoft/autogen#4014] [Q&A]
Hello, does Autogen 0.2 support GPT-4o (on Azure API)? I get openai.NotFoundError : DeploymentNotFound when I try. It works fine with the GPT-4 model deployed in the exact same way.

**Comment by ekzhu** [comment]
Yes. it does. DeploymentNotFound means your deployment was not found. Did you deploy the model?

**Reply by LasseTS** [reply]
Thank you! I was using a resource in Canada East. I set up a new resource in EastUS along with a new GPT4o model deployment, and that works!
